### PR TITLE
Added a bit of documentation

### PR DESCRIPTION
**Pull request by bebenlebricolo**
Added a small bit of documentation for packages installation and virtual environment setup (very basic stuff).

PS: Nice example, I landed on your page after looking for network graph vizualization tools and wanted to run the examples myself :smile:.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+dash
```
Comment (MohitMayankOP) [inline requirements.txt]: Can you plz add the versions here?

**Comment by MohitMayankOP** [comment]
Hey @bebenlebricolo thanks for the PR, really appreciate this. Btw reviewed the PR and left one comment, if you can resolve that I will merge it.

File: requirements.txt (hunk 1 above)
Comment (bebenlebricolo) [reply requirements.txt]: Yes sure ; I used the latest version for testing purposes on my machine and it worked straight out of the box. Some deprecation warnings though with visdcc, I'll stick with the latest stable releases

Comment (MohitMayankOP) [reply requirements.txt]: Sure

Comment (bebenlebricolo) [reply requirements.txt]: Should be ok by now, I let you review the latest patch and off we go ! :)

**Review by MohitMayankOP — APPROVED** [review]

**Pull request merged**